src/rpdk/core/data/managed-upload-infrastructure.yaml

```diff
@@ -41,34 +41,6 @@ Resources:
       VersioningConfiguration:
         Status: Enabled
 
-  ArtifactCopyPolicy:
-    Type: AWS::S3::BucketPolicy
-    Properties:
-      Bucket: !Ref ArtifactBucket
-      PolicyDocument:
-        Version: "2012-10-17"
-        Statement:
-          - Sid: Allow CloudFormation to copy artifacts from the bucket
```

Comment:
We should only remove this permission. But keep the `Require Secure Transport` deny statement, this is one of required permission for S3 bucket policy to secure the s3 bucket.